scripts/what-the-fips/fix.js

```diff
@@ -7,7 +7,7 @@ const promisify = require('util').promisify;
 const csvParse = promisify(require('csv-parse'));
 
 const POPULATION_CSV_URL =
-  'https://raw.githubusercontent.com/covid-projections/covid-data-model/main/libs/datasets/sources/fips_population.csv';
+  'https://raw.githubusercontent.com/act-now-coalition/covid-data-model/main/libs/datasets/sources/fips_population.csv';
```

Comment:
Currently this leads to a 404, whether we use `covid-projections` or `act-now-coalition` in the URL. I'm going to try and locate this file.

Reply:
Found it. It's in `https://raw.githubusercontent.com/act-now-coalition/covid-data-model/main/data/misc/fips_population.csv`. Will update now.